src/assets/modal-background.svg

```diff
@@ -1 +1,38 @@
-<svg width="726" height="64" fill="none" xmlns="http://www.w3.org/2000/svg"><mask id="a" maskUnits="userSpaceOnUse" x="0" y="0" width="726" height="64"><path d="M0 12C0 5.373 5.373 0 12 0h702c6.627 0 12 5.373 12 12v52H0V12z" fill="url(#paint0_linear)"/></mask><g mask="url(#a)"><path d="M726 0H0v64h726V0z" fill="#CBF3EF"/><path opacity=".6" d="M726 0H0v64h726V0z" fill="#7CE0D6"/><mask id="b" maskUnits="userSpaceOnUse" x="0" y="0" width="726" height="64"><path d="M0 12C0 5.373 5.373 0 12 0h702c6.627 0 12 5.373 12 12v52H0V12z" fill="url(#paint1_linear)"/></mask><g mask="url(#b)"><path d="M726 0H0v64h726V0z" fill="#8DE4DB"/><mask id="c" maskUnits="userSpaceOnUse" x="19" y="-1681" width="2212" height="2212"><path transform="rotate(135 2230.45 -1418.39)" fill="url(#paint2_linear)" d="M2230.45-1418.39h2756.09v371.382H2230.45z"/></mask><g mask="url(#c)"><path d="M2004.67-1612.62L95.271 296.776l207.3 207.3L2211.97-1405.32l-207.3-207.3z" fill="#CBF3EF"/><path opacity=".4" d="M2004.67-1612.62L95.271 296.776l207.687 207.687L2212.35-1404.93l-207.68-207.69z" fill="#7CE0D6"/><path opacity=".4" d="M2109.87-1507.42l-105.2-105.2L95.271 296.776l105.201 105.201L2109.87-1507.42z" fill="#7CE0D6"/><path opacity=".4" d="M2004.67-1612.62L95.271 296.776l65.751 65.751L2070.42-1546.87l-65.75-65.75z" fill="#7CE0D6"/></g></g></g><defs><linearGradient id="paint0_linear" x1="401.985" y1="-110.663" x2="394.944" y2="61.537" gradientUnits="userSpaceOnUse"><stop stop-color="#32FFF5"/><stop offset="1" stop-color="#01BFE3"/></linearGradient><linearGradient id="paint1_linear" x1="401.985" y1="-110.663" x2="394.944" y2="61.537" gradientUnits="userSpaceOnUse"><stop stop-color="#32FFF5"/><stop offset="1" stop-color="#01BFE3"/></linearGradient><linearGradient id="paint2_linear" x1="5053.71" y1="-1047.01" x2="2167.37" y2="-1067.44" gradientUnits="userSpaceOnUse"><stop offset=".14"/><stop offset="1" stop-opacity=".2"/></linearGradient></defs></svg>
\ No newline at end of file
+<svg width="726" height="64" viewBox="0 0 726 64" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
We forgot to optimize the asset

Reply:
Oh good catch, forgot to do that on a few others as well. I'll do it for all of them on a new PR.